core/client/src/main/java/org/phenotips/remote/client/internal/RemoteMatchFinder.java

```diff
@@ -78,6 +92,36 @@
     @Inject
     private ConsentManager consentManager;
 
+    @Inject
+    private Provider<XWikiContext> provider;
+
+    private XWikiDocument prefsDoc;
+
+    @Override
+    public void initialize() throws InitializationException
+    {
+        this.prefsDoc = this.getMatchingRunInfoDoc();
+    }
+
+    private XWikiDocument getMatchingRunInfoDoc()
+    {
+        try {
+            XWikiContext context = this.provider.get();
+            XWikiDocument doc = context.getWiki().getDocument(MATCHING_RUN_INFO_DOCUMENT, context);
+
+            if (doc != null && !doc.isNew()) {
+                return doc;
+            } else {
+                this.logger.error("Failed to retrieve matching run info document");
+            }
+
+        } catch (XWikiException e) {
+            this.logger.error("Failed to modify matching run info document: {}", e.getMessage(), e);
```

Comment:
the document is never modified, so the message is wrong (should be "failed to find" I suppose?)